controls/pcidss_4_ocp4.yml

```diff
@@ -2205,8 +2215,12 @@ controls:
         confirmed.
       levels:
         - base
-      status: pending
-      rules: []
+      status: not applicable
+      notes: |-
+        Account lockout for failed attempts are managed by the identity provider as all
```

Comment:
This is only true if an identity provider is used *and* kubeadmin is removed. I would say we could include those rules here, but it would be redundant with their usage elsewhere in the profile.

Could or should we add them as related rules?

Reply:
I think you comment could apply to most of requirement 8, so I added it to Sections 8.2 and 8.3, which have the core of requirements towards identity providers.

> I would say we could include those rules here, but it would be redundant with their usage elsewhere in the profile.

It should be fine for a control to have rules that are already selected by another control. The control should be complete and self-sufficient on their rule selections.

Reply:
Hmm, if one deploys OpenShift without selecting an identity provider, is he stuck with only the `kubeadmin` user?

Reply:
Yes - I believe that is the case.

Reply:
Strange - are we not including `idp_is_configured` anywhere in this profile, yet?

Reply:
We are, I see it in `8.2`, `8.2.1` and `8.3`.